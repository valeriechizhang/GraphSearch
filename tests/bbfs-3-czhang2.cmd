--energy 20000 --end-x=0 --end-y=6