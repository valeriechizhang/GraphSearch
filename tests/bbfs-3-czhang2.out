Solution steps: ['N', 'N', 'N', 'N', 'N', 'N']
Solution cost: 16026
Number of states considered: 12

Frontier:
Pos=(3, 0) Moves=['E', 'E', 'E'] Cost=3
Pos=(2, 1) Moves=['E', 'E', 'N'] Cost=4764
Pos=(1, 2) Moves=['E', 'N', 'N'] Cost=6323
Pos=(0, 3) Moves=['N', 'N', 'N'] Cost=8013
Pos=(0, 3) Moves=['N', 'N', 'N'] Cost=8013
Pos=(1, 4) Moves=['N', 'N', 'W'] Cost=6323
Pos=(2, 5) Moves=['N', 'W', 'W'] Cost=72
Pos=(3, 6) Moves=['W', 'W', 'W'] Cost=3

Closed List:
Pos=(0, 0) Moves=[] Cost=0
Pos=(0, 1) Moves=['N'] Cost=7922
Pos=(1, 0) Moves=['E'] Cost=1
Pos=(0, 2) Moves=['N', 'N'] Cost=8012
Pos=(1, 1) Moves=['E', 'N'] Cost=6243
Pos=(2, 0) Moves=['E', 'E'] Cost=2
Pos=(0, 6) Moves=[] Cost=0
Pos=(1, 6) Moves=['W'] Cost=1
Pos=(0, 5) Moves=['N'] Cost=90
Pos=(2, 6) Moves=['W', 'W'] Cost=2
Pos=(1, 5) Moves=['N', 'W'] Cost=81
Pos=(0, 4) Moves=['N', 'N'] Cost=8012
